source/command-monitoring/tests/unified/unacknowledgedBulkWrite.yml

```diff
@@ -0,0 +1,56 @@
+description: "unacknowledged-bulkWrite"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+      collectionOptions:
+        writeConcern: { w: 0 }
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+
+tests:
+# On a 2.4 server, no GLE is sent and requires a client-side manufactured reply
```

Comment:
this comment is copied from the original test. I'm not sure if it's relevant for anyone anymore though

Reply:
I'm in favor of dropping this comment. If you'd like to keep it, I'd propose moving it above the `commandSucceededEvent` line down below.